<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(205,240,255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
boder-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="categoria">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <item>
      <property name="text">
       <string>Selecione</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SSD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Memória RAM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Placa mãe</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mouse</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Placa de Rede</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cabo de rede</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>teclado</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Monito</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gabinete</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pendrive</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fonte de alimentação</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cabo VGA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cabo HDMI</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>191</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Garenciamento cadastro de produtos</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="botao_pesquisa">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>530</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 0, 127);
	color: rgb(255, 255, 255);
border-radius: 20px;
}
QPushButton:hover
{
   background-color:white;
	color: rgb(0, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Pesquisa</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pesquisa">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Logomarca (1).png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>731</width>
      <height>671</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Button_Cadastrar_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>560</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 0, 127);
	color: rgb(255, 255, 255);
border-radius: 20px;
}
QPushButton:hover
{
   background-color:white;
	color: rgb(0, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Alterar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Cadastrar_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>560</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 0, 127);
	color: rgb(255, 255, 255);
border-radius: 20px;
}
QPushButton:hover
{
   background-color:white;
	color: rgb(0, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Deleta</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Cadastrar_4">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>560</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 0, 127);
	color: rgb(255, 255, 255);
border-radius: 20px;
}
QPushButton:hover
{
   background-color:white;
	color: rgb(0, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Voltar</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>711</width>
       <height>421</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>cod</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nome</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>categoria</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>fornecedor</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>custo_fabrica</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>funcionario</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>valor_venda</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
